types/amphtml-validator/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please just use the default settings (entire file should be `{ "extends": "dtslint/dt.json" }`) -- I don't like semicolons either but it's better for the repository to stay consistent.

Reply:
Updated tslint & re-linted the new code.